### DISCUSSION TITLE
Restic Not Storing Data In S3

### DISCUSSION THREAD

**Discussion by dcplaya** [vmware-tanzu/velero#4233] [Community support Q&A]
I have a Velero instance running that is connected to an S3 (MinIO) backend. When Velero creates a backup, the backup gets stored in the S3 instance but no Restic data is being store. The restic pods also don't show any errors about unable to store the data. 

This install was previously working with a different instance of MinIO. I have since moved to a fresh install of MinIO, the only thing that has changed was the URL, which was updated in the helm chart. I know the URL works because the Velero data is being stored there, just not the Restic data.

How do I go about debugging this?

**Comment by billmetangmo** [comment]
Hi @dcplaya  , do you add **--default-volumes-to-restic** to your velero backup command ?

**Comment by dcplaya** [comment]
@billmetangmo 
I actually annotate each PVC I want so I dont backup everything (mainly because I am still not 100% sure it wont try and backup an NFS share)

But I finally fixed my issue. Moving S3 buckets seems to cause issues. I had to remove the Velero helm release (I delete the namespace), remove the bucket. Then recreate the bucket and re-apply the helm release.

I am not sure what hangs around that makes Restic not use the new S3 bucket but at least I have a work around.
